PowervRA/Functions/Public/config-service/Get-vRANode.psm1

```diff
@@ -0,0 +1,141 @@
+function Get-vRANode {
+<#
+    .SYNOPSIS
+    List all vRA nodes registered to the appliance.
+    
+    .DESCRIPTION
+    List all vRA nodes registered to the appliance.
+
+    .PARAMETER Name
+    The name of the node.
+
+    .PARAMETER Role
+    The roles installed on the node.
+    
+    .INPUTS
+    System.String
+
+    .OUTPUTS
+    System.Management.Automation.PSObject.
+
+    .EXAMPLE 
+    Get-vRANode -Name vra-01a.lab.local
+
+    .EXAMPLE
+    Get-vRANode -Role Website
+
+#>
+[CmdletBinding(DefaultParameterSetName="Standard")][OutputType('System.Management.Automation.PSObject')]
+
+    Param (
+    
+    [parameter(Mandatory=$false,ParameterSetName="ByName")]
+    [ValidateNotNullOrEmpty()]
+    [String]$Name,
```

Comment:
Worth considering accepting multiple strings here? So that you could do something like:

Get-vRANode -Name vra-01a.lab.local, vra-02a.lab.local

Take a look at something like Get-vRABlueprint to so how we normally implement something like that, i.e.:

[String[]] by the parameter name, then a foreach loop in the code